clients/hadoopfs/src/test/java/io/lakefs/contract/TestLakeFSFileSystemContract.java

```diff
@@ -51,7 +47,15 @@ protected Path path(String pathString) {
   }
 
   public void testRenameFileAsExistingFile() throws Exception {
-    intercept(FileAlreadyExistsException.class, super::testRenameFileAsExistingFile);
+    if (!renameSupported()) return;
```

Comment:
Why do we add a condition here?  Don't we know whether or not it should be supported?

Reply:
good call - I'll remove this code. basically, we copied the s3a contract implementation, but for us, we can keep it clear and clean.

Reply:
removed - it is just a copy of the s3a implementation